spread.yaml

```diff
@@ -336,14 +341,17 @@ suites:
             if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
                 . $TESTSLIB/pkgdb.sh
                 distro_purge_package snapd
-                distro_purge_package snap-confine
+                if [[ "$SPREAD_SYSTEM" != opensuse-* ]]; then
+                    # A snap-confine package never existed on openSUSE
+                    distro_purge_package snap-confine
+                fi
             fi
 
     tests/completion/:
         summary: completion tests
 
         # ppc64el disabled because of https://bugs.launchpad.net/snappy/+bug/1655594
-        systems: [-ubuntu-core-*, -ubuntu-*-ppc64el, -fedora-*]
+        systems: [-ubuntu-core-*, -ubuntu-*-ppc64el, -fedora-*, -opensuse-*]
```

Comment:
Why are those tests disabled on Fedora and openSUSE? Can you please add a comment.